types/eslint/index.d.ts

```diff
@@ -773,6 +773,19 @@ export namespace Rule {
     }
 }
 
+interface ObjectMetaProperties {
```

Comment:
```suggestion
export interface ObjectMetaProperties {
```

For reuse?

Reply:
I was leaving it unexported because it didn't fit neatly into one of the existing namespaces, and was really just a helper to inject these props into other interfaces - it didn't seem like a useful enough thing for consumers to want to reference directly?

Reply:
https://github.com/mdx-js/eslint-mdx/pull/468/files#diff-df958e7e9a25ffa41030787a1ae0f67a5a864b0709f0298d93a20a6bdf298045R5

I'd like to use `const meta: ObjectMetaProperties`

Reply:
OK, nevermind, the linter is forcing me to anyway, since it's implicitly exported. I've moved it into the `ESLint` namespace so we don't have a random utility interface exposed at the root of the package.